helper/config.js

```diff
@@ -20,6 +20,15 @@ function readConfig(window) {
   }
 }
 
+function resetConfig(window) {
+  try {
+    store.delete('config');
+    window.webContents.send('configErased');
+  } catch {
```

Comment:
Does that throw any useful error info?

Reply:
Truthfully, we should never actually get to that `catch`, it's just in place in case something is seriously wrong!